Flaky test  TestBitmap

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-kvrocks/issues) and found no similar issues.


### Version

unstable

### Minimal reproduce step

TestBitMap fails in https://github.com/apache/incubator-kvrocks/actions/runs/5070341119/jobs/9105869084?pr=1444

### What did you expect to see?

No error.

### What did you see instead?

```
--- FAIL: TestBitmap (355.95s)
    --- FAIL: TestBitmap/SETBIT/GETBIT/BITCOUNT/BITPOS_boundary_check_(type_bitmap) (22.25s)
        bitmap_test.go:149: 
            	Error Trace:	/Users/runner/work/incubator-kvrocks/incubator-kvrocks/tests/gocase/unit/type/bitmap/bitmap_test.go:149
            	Error:      	Not equal: 
            	            	expected: 4294967295
            	            	actual  : 0
            	Test:       	TestBitmap/SETBIT/GETBIT/BITCOUNT/BITPOS_boundary_check_(type_bitmap)
```

### Anything Else?

_No response_

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

--- a/tests/gocase/unit/type/bitmap/bitmap_test.go
+++ b/tests/gocase/unit/type/bitmap/bitmap_test.go
@@ -140,13 +140,22 @@ func TestBitmap(t *testing.T) {
 	})
 
 	t.Run("SETBIT/GETBIT/BITCOUNT/BITPOS boundary check (type bitmap)", func(t *testing.T) {
-		require.NoError(t, rdb.Del(ctx, "b0").Err())
+		cmd := rdb.Del(ctx, "b0")
+		require.NoError(t, cmd.Err())
 		var maxOffset int64 = math.MaxUint32
-		util.ErrorRegexp(t, rdb.SetBit(ctx, "b0", maxOffset+1, 1).Err(), ".*out of range.*")
-		require.NoError(t, rdb.SetBit(ctx, "b0", maxOffset, 1).Err())
-		require.EqualValues(t, 1, rdb.GetBit(ctx, "b0", maxOffset).Val())
-		require.EqualValues(t, 1, rdb.BitCount(ctx, "b0", &redis.BitCount{Start: 0, End: maxOffset / 8}).Val())
-		require.EqualValues(t, maxOffset, rdb.BitPos(ctx, "b0", 1).Val())
+		cmd = rdb.SetBit(ctx, "b0", maxOffset+1, 1)
+		util.ErrorRegexp(t, cmd.Err(), ".*out of range.*")
+		cmd = rdb.SetBit(ctx, "b0", maxOffset, 1)
+		require.NoError(t, cmd.Err())
+		cmd = rdb.GetBit(ctx, "b0", maxOffset)
+		require.NoError(t, cmd.Err())
+		require.EqualValues(t, 1, cmd.Val())
+		cmd = rdb.BitCount(ctx, "b0", &redis.BitCount{Start: 0, End: maxOffset / 8})
+		require.NoError(t, cmd.Err())
+		require.EqualValues(t, 1, cmd.Val())
+		cmd = rdb.BitPos(ctx, "b0", 1)
+		require.NoError(t, cmd.Err())
+		require.EqualValues(t, maxOffset, cmd.Val())
 	})
 
 	t.Run("BITOP NOT (known string)", func(t *testing.T) {

